### PR TITLE
Sphinx docs

### PR DESCRIPTION
**Pull request by sndrtj**

### CODE REVIEW

File: docs/source/classification.rst [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+::
+
+    rna_cd-classify -m model.json -l bams.list -j 3 -c chrM \
+    --chunksize 100 -o classifications.out
+
+
+Usage
+-----
```
Comment (rhpvorderman) [inline docs/source/classification.rst:67]: use sphinx-argparse for this. You will not regret it.

File: docs/source/training.rst [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+
+::
+
+    rna_cd-train -c chrM -pl positives.list -nl negatives.list -j 3 \
+    --chunksize 100 -o model.json --plot-out pca.png
+
+
+Usage
+-----
+
```
Comment (rhpvorderman) [inline docs/source/training.rst:70]: spinhx-argparse!

File: requirements-dev.txt [hunk 3]
```diff
@@ -1,4 +1,5 @@
 pytest>=4.1.0
 flake8>=3.6.0
 pytest-cov>=2.6.1
-python-magic>=0.4.15
\ No newline at end of file
+python-magic>=0.4.15
+sphinx>=1.8.4
```
Comment (rhpvorderman) [inline requirements-dev.txt]: and sphinx-argparse :wink:

**Review by rhpvorderman — APPROVED** [review]

File: docs/source/classification.rst (hunk 1 above)
Comment (sndrtj) [reply docs/source/classification.rst:67]: Great, didn't know this existed! I'm using click in stead of argparse, but turns out there is a sphinx-click extension as well! xD

**Pull request merged**